src/harness/testcases/grant_testcase.py

```diff
@@ -433,17 +460,17 @@ def test_WINFF_FT_S_GRA_22(self):
     self.assertTrue(response['response']['responseCode'] in (202, 103))
 
   @winnforum_testcase
-  def test_WINFF_FT_S_GRA_23(self):
+  def test_WINNF_FT_S_GRA_23(self):
     """Dual grant requests for two devices. Successful case.
 
     The response should be 0 (NO_ERROR)
     """
     # Register the devices
     device_a = json.load(
-        open(os.path.join('testcases', 'testdata', 'device_a.json')))
-    device_b = json.load(
-        open(os.path.join('testcases', 'testdata', 'device_b.json')))
-    request = {'registrationRequest': [device_a, device_b]}
+      open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    device_c = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_c.json')))
+    request = {'registrationRequest': [device_a, device_c]}
     response = self._sas.Registration(request)['registrationResponse']
```

Comment:
Looks like this test case (and several others) are missing a call to InjectFccId(). If you have time, maybe you could do this in your next PR?